tests/integration/targets/test_mysql_replication/tasks/mysql_replication_initial.yml

```diff
@@ -8,6 +8,26 @@
       login_host: 127.0.0.1
 
   block:
+    - name: find out the database version
+      mysql_info:
+        <<: *mysql_params
+        login_port: '{{ mysql_primary_port }}'
+        filter: version
+      register: db
+
+    - name: Set mysql8022_and_higher
+      set_fact:
+        mysql8022_and_higher: false
+
+    - name: Set mysql8022_and_higher
+      set_fact:
+        mysql8022_and_higher: true
+      when:
+      - db.version.major > 8 or (db.version.major == 8 and db.version.minor > 0) or (db.version.major == 8 and db.version.minor == 0 and db.version.release >= 22)
+
+    - name: Debug mysql8022_and_higher
+      debug:
+        msg: "{{ mysql8022_and_higher }} , {{ db.version }}"
```

Comment:
```suggestion
        msg: "{{ mysql8022_and_higher }}, {{ db.version }}"
```

Reply:
i removed it at all:) It's there from debugging times, forgot to remove